simtools/util/general.py

```diff
@@ -147,13 +145,16 @@ def validate_config_data(config_data, parameters, ignore_unidentified=False):
         if par_name in out_data:
             continue
         if "default" in par_info.keys() and par_info["default"] is not None:
-            validated_value = _validate_and_convert_value(par_name, par_info, par_info["default"])
+            default_value = par_info["default"]
+            if not isinstance(default_value, u.Quantity) and "unit" in par_info:
+                default_value *= par_info["unit"]
+            validated_value = _validate_and_convert_value(par_name, par_info, default_value)
```

Comment:
You are absolutely right and I was aware of it (that's why I implemented here the missing tests). I didn't continue because there are more urgent things to do. I will open a test for it so it's not forgotten.